liquigraph-core/src/test/java/org/liquigraph/core/io/ChangelogGraphWriterTestSuite.java

```diff
@@ -297,4 +339,9 @@ private static Object property(Object changeset, String name) {
         assertThat(changeset).isInstanceOf(Node.class);
         return ((Node)changeset).getProperty(name);
     }
+
+    private void given_inserted_data(String query, Connection connection) throws SQLException {
+        connection.createStatement().executeQuery(query);
+        connection.commit();
```

Comment:
By the way, that simple `commit()` is the fix for the failure of the unit test I had during the Hackergarten. I could have sworn I'd tested it at the time, though.